modules/tensor_mechanics/src/materials/ComputeAxisymmetricRZFiniteStrain.C

```diff
@@ -34,7 +34,7 @@ ComputeAxisymmetricRZFiniteStrain::initialSetup()
 }
 
 Real
-ComputeAxisymmetricRZFiniteStrain::computeDeformGradZZ()
```

Comment:
I don't think these functions make any sense for axisymmetric cases. The out of plane strain (theta theta) is computed as a function of the radial displacement.